### PR TITLE
Add tarot cards and Ouija board

### PR DESCRIPTION
**Pull request by aapocalypseAArisen**
#### Summary
Content "Added tarot cards as a variant of playing cards and added Ouija board as a playable board game."

#### Purpose of change
I noticed a lack of mystically inclined games and decided to add them. 

#### Describe the solution

Added these games into Cataclysm.

#### Describe alternatives you've considered

Not having these games. Nothing horrible would happen but it would be an opportunity not taken.

#### Testing

Used the debug menu to spawn in both the tarot cards and Ouija board. Successfully played them like any other game.

#### Additional context

First time contributing anything. We'll see how it goes.

### CODE REVIEW

File: data/json/items/fluff.json [hunk 1]
```diff
@@ -61,6 +61,12 @@
         "description": "A set of cards meant to play the game \"Sorcery.\"  Each card has a fun picture of a different monster.",
         "weight": 2
       },
+      {
+        "id": "deck_of_tarot_cards",
+        "name": { "str": "deck of tarot cards", "str_pl": "decks tarot of cards" },
```
Comment (Qrox) [inline data/json/items/fluff.json:66]:
```suggestion
        "name": { "str": "deck of tarot cards", "str_pl": "decks of tarot cards" },
```

File: data/json/items/fluff.json [hunk 2]
```diff
@@ -61,6 +61,12 @@
...
+        "description": "A deck of tarot cards. Commonly used for the practice of divination. Each card has a unique and highly symbolic picture.",
```
Comment (Qrox) [inline data/json/items/fluff.json:67]:
```suggestion
        "description": "A deck of tarot cards.  Commonly used for the practice of divination.  Each card has a unique and highly symbolic picture.",
```

File: data/json/items/fluff.json [hunk 3]
```diff
@@ -640,6 +646,23 @@
     "flags": [ "NO_REPAIR" ],
     "use_action": [ "PLAY_GAME" ]
   },
+  {
+    "id": "ouija",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Ouija board" },
+    "description": "A wooden board with the alphabet and numbers on its face. Commonly used for communing with spirits of the dead. A planchette is included in the set.",
```
Comment (Qrox) [inline data/json/items/fluff.json:654]:
```suggestion
    "description": "A wooden board with the alphabet and numbers on its face.  Commonly used for communing with spirits of the dead.  A planchette is included in the set.",
```

Comment (aapocalypseAArisen) [reply data/json/items/fluff.json]: Thanks.

**Comment by PGR-14** [comment]
Maybe we could add something in w/ portal storms?

**Comment by aapocalypseAArisen** [comment]
Am I missing anything that would allow merging?

**Pull request merged**